ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -508,6 +508,34 @@ In AVD 4.0.0, we had "switched" as the default value for `ethernet_interfaces[].
 
 With AVD 5.0.0, the default value for `type` in `ethernet_interfaces` and `port_channel_interfaces` is no longer supported. The `type` key must now be explicitly defined in the input variables if it is needed in the configuration and documentation.
 
+### Removal of `type` key dependency for rendering ethernet/port-channel interfaces configuration and documentation
+
+#### IPv4 and IPv6 tables
```

Comment:
I don't think we need a section for the documentation tables on their own.

Reply:
Fixed